galaxy/export_user_files.py

```diff
@@ -48,10 +48,14 @@ def change_path( src ):
 
     galaxy_root_dir = os.environ.get('GALAXY_ROOT', '/galaxy-central/')
 
-    if os.path.exists( '/export/.distribution_config/' ):
-        shutil.rmtree( '/export/.distribution_config/' )
-    shutil.copytree( os.path.join(galaxy_root_dir, 'config'), '/export/.distribution_config/' )
-
+    galaxy_distrib_paths = {'/galaxy-central/config/': '/export/.distribution_config',
+                            '/galaxy-central/lib': '/export/galaxy-central/lib',
+                            '/galaxy-central/.venv': '/export/galaxy-central/.venv',
```

Comment:
Yeah, I agree. For now this depends on the galaxy models, I think it'll be a while before/if ever galaxy-lib can do this.
The nice thing is that at least the lib and .venv are in sync with galaxy in the container.